guides/v2.1/cloud/live/stage-prod-migrate-prereq.md

```diff
@@ -20,7 +20,7 @@ This information is broken down into prerequisite steps for [Starter](#starter)
 
 ## Starter plan projects {#starter}
 
-For Starter, make sure all of your code is merged into a single development branch to test prior to deployment to Production. Due to all of your branches and environments being on a PaaS infrastructure, you only need access information for Master Production and other environments as needed. This access information includes your store URLs and SSH link.
+For Starter, verify all of your code merged into a single development branch to test prior to deployment to Production. Due to all of your branches and environments being on a PaaS infrastructure, you only need access information for Master Production and other environments as needed. This access information includes your store URLs and SSH link.
```

Comment:
verify that ... However, the sentence doesn't quite make sense --something missing

Reply:
I felt compelled to fix this. Darn rabbits. Does this help?
"For Starter, merge all code into one "staging" branch for testing prior to deploying to Production. All active branches and environments are part of the PaaS infrastructure and have access to information, such as store URLs and SSH link."